Manuel/Challenges/buy_me_a_laptop_challenge.py

```diff
@@ -0,0 +1,111 @@
+# Challenge 1
+print("Challenge 1: All possible laptops\n")
+
+print("Question 1: You are given a list containing the laptop names. Print the names of each the laptops separately.")
+all_laptops = ["Apple MacBook Pro", "Asus Zenbook", "Dell XPS", "Lenovo IdeaPad", "Apple MacBook Air", "Sony Viao"]
+
+
+# TODO: Write code to print all laptop names
+print(all_laptops[0])
+print(all_laptops[1])
+print(all_laptops[2])
+print(all_laptops[3])
+print(all_laptops[4])
+print(all_laptops[5])
+
```

Comment:
Please redo this section using a for-loop. You can accomplish in 2 lines what you wrote in 6 lines here. A major part of writing the most efficient code is to do it in the least lines as possible.